packages/block/utilities/api/getBlockProps.js

```diff
@@ -0,0 +1,5 @@
+export const getBlockProps = ({ blockId, height }) => {
```

Comment:
please rename directory to utils
also for this api file please create a directore called api and move them there